manuscript.Rmd

```diff
@@ -424,30 +457,73 @@ In that case, the readers can be certain, that they have received a complete acc
-Note that any single study that does not produce evidence in favor of a research hypothesis is still precious for the scientific process.
-Under the current incentive structure, however, many researchers have reason to avoid such outcomes and few, if any, have the privilege to operate outside of these incentives.
-
-In our view, there are two advantages to focusing on uncertainty over theoretical risk itself.
-First, aiming to reduce uncertainty is beneficial across a wide range of potential theoretical risks one encounters as researcher.
-That is, researchers benefit from preregistering their study, regardless of whether or not the study is tagged "exploratory".
-If researchers conduct a more exploratory study they can clearly communicate how exploratory they aim to be and their results can be judged accordingly.
-Second, if researchers realize after the fact that a deviation from their preregistration in certain details is sensible (and sometimes, the only reasonable way forward), they are explicitly allowed to change their strategy.
-<!--AB: this sounds like a new claim but it is accepted that deviations are OK, isn't it? maybe stress that this view is natural under the proposed framework but less so in stricter views of preregistration-->
-Allowing researchers to deviate from the preregistration is controversial (XXX) <!--AB: ref missing--> because the authors might sift through all the possibilities and then select the most beneficial for their theory.
-In its full consequence, this argument might result in the same uncertainty that we argued plagues not-preregistered studies.
-However, if authors of study offer a convincing argument for their deviation, their readers might may judge it to be a reasonable explanation for the deviation.
-In fact, using the here presented framework, we can distinguish reasonable from unreasonable deviations.
-If a deviation increases detectability enough to offset the increase in uncertainty in theoretical risk, it is a deviation worth doing.
+To summarize, we showed that higher theoretical risk and lower uncertainty about theoretical risk lead to higher expected epistemic value across a variety of measures.
+The former result, that increasing theoretical risk leads to higher expected epistemic value, reconstructs the appeal of confirmatory research.
+However, theoretical risk is something researcher have only limited control over, e.g., theories are often vague and illdefined, resources are limited, and increasing theoretical risk usually decreases detectability (a special instance of this trade off is the well known tension between alpha error and power).
+Before researchers may preregister a confirmatory study with high theoretical risk, they must ensure that they are able to conduct a confirmatory study.
```

Comment:
```suggestion
While we believe that preregistration is always beneficial, it might be counterproductive to pursue high theoretical risk if the conditions for highly confirmatory research are not given.
```